lib/stat_tracker.rb

```diff
@@ -23,6 +23,48 @@ def initialize(files)
     end
   end
 
+  #array off all gameteam objects played in that season
+  #can't get from just @games because doesn't have tackles so need to relate
+  #season_id to game id then game_id in gameteams data to game_id there
+  #this can definitely be refactored
+  def games_played_in_season(season)
+    game_objects_by_season = @games.group_by do |game|
+      game.season
+    end
+    get_games_in_season = game_objects_by_season.keep_if do |key, value|
+      key == season
```

Comment:
Optional refactoring idea: as questioning if keep_if is necessary. I believe we can combine the grouping and the conditional logic of key == season to one method and reduce hash iterations.

Reply:
yeah for sure. Will work on that and push up before we merge.